scripts/migrate-02_0.3-to-0.5.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2017 Redlink GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+var smarti = db.getCollection('smarti'),
```

Comment:
@ja-fra Just an idea: moving this common code to some sort of framework - this might be possible to be built by Maven, so that each file was executable on its own